scripts/cluster/setup_master_node.sh

```diff
@@ -57,4 +60,18 @@ fi
 kubectl apply --filename $ROOT/configs/knative_yamls/serving-default-domain.yaml
 
 kubectl apply --filename https://github.com/knative/net-istio/releases/download/v0.19.0/release.yaml
+
```

Comment:
no, this part cannot be on by default because tracing may result in a slowdown. You should have it as a separate script.

Reply:
hmm, you are right.